src/index.js

```diff
@@ -31,6 +36,38 @@ ReactDOM.render(
           selection="1"
         />
       </div>
+
+      <div className="App" style={{ width: '100vw', height: '100vh' }}>
+        <Find text="increase in serious accidents by driving on underinflated tires" image="FlatTire" selection="1" />
+      </div>
+
+      <div className="App" style={{ width: '100vw', height: '100vh' }}>
+        <Location
+          text="increase in serious accidents by driving on underinflated tires"
+          image="FlatTire"
+          selection="1"
```

Comment:
The text, image and selection properties are unused, and should not be passed down as properties. This goes for every page.